roles/postgresql/defaults/main.yml

```diff
@@ -1,11 +1,25 @@
 ---
 # copyright Utrecht University
 
+# version of PostgreSQL to install (tested with 11-14)
+pgsql_version: 11
+prevent_upgrade: false
```

Comment:
This seems like an unsafe default (at least for acc and production environments). From an admin perspective, would prefer to only initiate a database upgrade if that is explicitly configured.